src/entity/Entity.php

```diff
@@ -656,7 +657,10 @@ protected function entityBaseTick(int $tickDiff = 1) : bool{
 		}
 		$this->checkBlockIntersectionsNextTick = true;
 
-		if($this->location->y <= 0 && $this->isAlive()){
+		$cfg = $this->server->getConfigGroup();
+		$minY = $cfg->getPropertyInt(YmlServerProperties::LEVEL_SETTINGS_MIN_Y, 0);
+
+		if($this->location->y <= $minY && $this->isAlive()){
```

Comment:
Given this method will get called often it might be nice to load the minY into the world and using $world->getMinY() or something.

Reply:
I'll make it settable and public for developers to use if they need.

Reply:
Now that I think about it, I won't make it settable 😔